test/model_test.dart

```diff
@@ -518,7 +518,7 @@ void main() {
       expect(
           packageGraph
               .localPackages.first.defaultCategory.publicLibraries.length,
-          equals(7));
+          equals(8));
```

Comment:
Aside: is `equals` necessary here?  I see the matcher elided below for strings (`expect(package.name, 'test_package');`).  FWIW: I'm on the fence about the short-hand myself.  Needless to say, whatever floats your boat is cool with me! :)

Reply:
leaving the style for integers in place, for now.  a good idea though.